sycl/CMakeLists.txt

```diff
@@ -180,6 +180,8 @@ configure_file("${version_header}.in" "${version_header}")
 # Copy SYCL headers
 add_custom_target(sycl-headers ALL
 COMMAND ${CMAKE_COMMAND} -E copy_directory ${sycl_inc_dir}/CL ${SYCL_INCLUDE_BUILD_DIR}/CL
+COMMAND ${CMAKE_COMMAND} -E copy_directory ${sycl_inc_dir}/sycl ${SYCL_INCLUDE_BUILD_DIR}/sycl
```

Comment:
aren't these lines the same as https://github.com/intel/llvm/pull/1853/files#diff-ad9ca624b2a795a5f24145a17c806d1fR183?

Reply:
Yes, they are. Please review only 3rd commit:
[SYCL][ESIMD] Explicit SIMD math APIs.